### DISCUSSION TITLE
山狮帮落网了

### DISCUSSION THREAD

**Discussion by fairy-2024** [share121/inter-knot#2058] [其他]
![Snipaste_2024-08-14_12-32-23](https://github.com/user-attachments/assets/cad6e3db-4675-4763-94b3-c208b38529fc)

刚从可靠信源得到的消息。媒体还没动静，不过应该很快就会看到相关新闻了，毕竟算得上一件大案。
发帖是为了提醒某些同胞，如果以前替山狮帮提供过服务…最近还是低调点比较好

**Comment by yutong903** [comment]
天哪，媒体马上就报道了，就是没指名道姓罢了......
